js/app1.js

```diff
@@ -1,121 +1,225 @@
 'use strict';
 
-let count = 0;
-
-let quest1 = prompt('What is your name?');
-alert('Hey ' + quest1 + '! Nice to meet you!');
-//console.log(alert);
+// function functionName(parameter){
+//
+// }
+
+function askName(){
+  let ans = prompt('What is your name?');
+  alert('Hey ' + ans + '! Nice to meet you!');
+  return ans;
+}
 
-let askTravel = prompt('Do you like traveling?');
-askTravel = askTravel.toLowerCase();
+function askTrips(){
+  let askTravel = prompt('Do you like traveling?');
+  askTravel = askTravel.toLowerCase();
 
-if (askTravel === 'yes' || askTravel === 'y') {
-  //console.log(askTravel);
-  alert('I love traveling too!');
-} else if (askTravel === 'no' || askTravel === 'n') {
-  //console.log(askTravel);
-  alert('You should give it another try. Traveling is a lot of fun.');
+  if (askTravel === 'yes' || askTravel === 'y') {
+    //console.log(askTravel);
+    alert('I love traveling too!');
+  } else if (askTravel === 'no' || askTravel === 'n') {
+    //console.log(askTravel);
+    alert('You should give it another try. Traveling is a lot of fun.');
+  }
 }
 
-let askCountries = prompt('Can you guess how many countries I have lived in? 1-5.');
-if (askCountries <= '3') {
-  alert('You are incorrect! The answer is 4.');
-  count = count - 1;
-} else if (askCountries === '4') {
-  alert('You are correct!');
-  count = count + 1;
-} else if (askCountries === '5') {
-  alert('You went too far ' + quest1 + '! The asnwer is 4.');
-  count = count - 1;
+function askDestinations(counter){
+  let askCountries = prompt('Can you guess how many countries I have lived in? 1-5.');
+  if (askCountries <= '3') {
+    alert('You are incorrect! The answer is 4.');
+  } else if (askCountries === '4') {
+    alert('You are correct!');
+    counter = counter + 1;
+  } else if (askCountries >= '5') {
+    alert('You went too far ' + quest1 + '! The answer is 4.');
+  }
+  return counter;
```

Comment:
Great use of a return statement here!

Don't forget, when you want to increase something by 1 you can just use the ++ operator.